internal/plugin/server.go

```diff
@@ -183,11 +183,13 @@ func (plugin *nvidiaDevicePlugin) Serve() error {
 	go func() {
 		lastCrashTime := time.Now()
 		restartCount := 0
+		maxRestarts := 5
+		crashTimeoutSeconds := float64(3600) // 1 hour in seconds
 
 		for {
 			// quite if it has been restarted too often
 			// i.e. if server has crashed more than 5 times and it didn't last more than one hour each time
```

Comment:
This comment should be moved to the docstring or at the point where we declare the variables.